config/config.go

```diff
@@ -482,7 +484,9 @@ func mergeEnvironmentVariables(c *AgentConfig) *AgentConfig {
 		durationS, _ := strconv.Atoi(v)
 		c.ContainerCacheDuration = time.Duration(durationS) * time.Second
 	}
-
+	if v := os.Getenv("DD_PROCESS_AGENT_CONTAINER_SOURCE"); v != "" {
```

Comment:
Is there some configuration that can be shared with the main Agent? I've been trying to cut down on the number of config that's specifically for the process agent where possible.